docs/api-site-config.md

```diff
@@ -167,6 +167,8 @@ h1 {
 
 `wrapPagesHTML` - Boolean flag to indicate whether `html` files in `/pages` should be wrapped with Docusaurus site styles, header and footer. This feature is experimental and relies on the files being `html` fragments instead of complete pages. It inserts the contents of your `html` file with no extra processing. Defaults to `false`.
 
+`cleanUrl` - If `true`, allow URLs with no `html` extension. Example: request to URL https://docusaurus.io/docs/installation will returns the same result as https://docusaurus.io/docs/installation.html.
```

Comment:
Can you move this up to be alphabetical with the rest of the optional fields?

Reply:
I can. But i'm out of reach from my pc until this weekend. I'll do it later if you don't mind. Sorry!

Reply:
Oh - no worries. I can do it for you 👍